script.js

```diff
@@ -1,29 +1,285 @@
-let blurredAreaModal = document.querySelector('.modal-blurred-area')
 let showInstructionsButton = document.querySelector('#instruction-button')
-let instructionsModal = document.querySelector('.instruction-modal')
 let hideInstructionsModal = document.querySelector('#modal-button')
-let closeModal = (status) => {
-    if(status === true) {
-        blurredAreaModal.style.display = 'none'
-        instructionsModal.style.display = 'none'
-    } else if( status === false) {
-        blurredAreaModal.style.display = 'block'
-        instructionsModal.style.display = 'block'
+
+function openCloseModal(modalOverlay, modalBody, status) {
+    if(status === 'close') {
+        document.querySelector(modalOverlay).style.display = 'none'
+        document.querySelector(modalBody).style.display = 'none'
+    } else if(status === 'open') {
+        document.querySelector(modalOverlay).style.display = 'block'
+        document.querySelector(modalBody).style.display = 'block'
```

Comment:
rename `status` to `isModalOpen` and instead of strings use true and false for the values (as discussed with Ollie)